backend/src/main/java/com/google/minicrm/data/NotificationsFrequency.java

```diff
@@ -0,0 +1,53 @@
+package com.google.minicrm.data;
+
+import com.google.appengine.api.datastore.EmbeddedEntity;
+
+public final class NotificationsFrequency {
+    private boolean onEveryLead;
+    private boolean daily;
+    private boolean weekly;
+
+    public NotificationsFrequency(boolean onEveryLead, boolean daily, boolean weekly) {
+        this.onEveryLead = onEveryLead;
+        this.daily = daily;
+        this.weekly = weekly;
+    }
```

Comment:
Why don't we use an enum to implement this? Would be straightforward and consistent to keep the implementation as the LeadStatus enum.

Reply:
+1